mod-tool/src/main/scala/at/forsyte/apalache/tla/Tool.scala

```diff
@@ -93,19 +93,19 @@ object Tool extends LazyLogging {
         command match {
           case Some(parse: ParseCmd) =>
             val injector = injectorFactory(parse)
-            handleExceptions(injector, runParse(injector, parse))
+            handleExceptions(injector, runParse(injector, parse))(parse.file)
 
           case Some(check: CheckCmd) =>
             val injector = injectorFactory(check)
-            handleExceptions(injector, runCheck(injector, check))
+            handleExceptions(injector, runCheck(injector, check))(check.file)
 
           case Some(test: TestCmd) =>
             val injector = injectorFactory(test)
-            handleExceptions(injector, runTest(injector, test))
+            handleExceptions(injector, runTest(injector, test))(test.file)
 
           case Some(typecheck: TypeCheckCmd) =>
             val injector = injectorFactory(typecheck)
-            handleExceptions(injector, runTypeCheck(injector, typecheck))
+            handleExceptions(injector, runTypeCheck(injector, typecheck))(typecheck.file)
```

Comment:
Not really, since this is not an output file, but the input specification file passed by the user. It's used to read the specification string, to copy into the issue body.

Reply:
No, this file is the `example.tla` in e.g. `check example.tla`, which can be anywhere on the filesystem and doesn't get copied into the rundir at any point.

Reply:
I understand that. But the only reason you need to pass it in here is in order to use it for generating the bug report output, no? Anyhow, this isn’t a critical thing. Just seems a bit off to me to be passing in additional arguments here when we already have access to the data via the output manager, which we need anyhow in the bug report template generator.

Reply:
The thing is, the `OutputManager` doesn't know about this file, because it's not otherwise related to outputs.

Reply:
That seems sensible, but at the same time, probably scope creep for this particular PR. You should make an issue for it though, since I can see it being relevant again with the other part of #943, where the error explainer displays the contents of the input file line where the issue is, for which it will need access to the source file too.

Reply:
I think it should only a take a few lines of code to make the proposed change, but if you'd rather, perhaps leave a note on #1069 that this should be corrected as part of that work, and I can take care of it later this week?